chirpstack/src/config.rs

```diff
@@ -89,6 +90,31 @@ impl Default for Redis {
     }
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+#[serde(default)]
+pub struct Sqlite {
+    pub path: String,
+    pub max_open_connections: u32,
+    #[serde(default = "default_sqlite_busy_timeout")]
+    pub busy_timeout: u32,
```

Comment:
If it is a timeout, then a `Duration` type would be more appropriate. As well as the usage of `#[serde(with = "humantime_serde")]`. That way, users can specify `100ms`, `10s`, etc...